kinesis/testing.py

```
@@ -0,0 +1,546 @@
+
+    @property
+    def stream(self) -> MemoryStream:
+        return MockKinesisBackend.get_stream(self.stream_name)
+
+    async def __aenter__(self) -> "MockConsumer":
+        _ensure_stream(self.stream_name, self._create_stream, self._create_stream_shards)
+
+        stream = self.stream
+        for i, shard in enumerate(stream.shards):
+            # Allocate shard on checkpointer (returns last checkpointed sequence)
+            success, last_seq = await self.checkpointer.allocate(shard.shard_id)
```

Ruff (0.15.0)

[warning] Unpacked variable `success` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)